platform/ios/app/MBXViewController.m

```diff
@@ -908,6 +912,33 @@ - (void)styleDynamicPointCollection
     [self.mapView.style addLayer:layer];
 }
 
+- (void)styleCountryLabelLanguage
+{
+    NSString *bestLanguageForUser = [NSString stringWithFormat:@"{name_%@}", [self bestLanguageForUser]];
+    MGLSymbolStyleLayer *countryLayer = (MGLSymbolStyleLayer *)[self.mapView.style layerWithIdentifier:@"country-label-lg"];
```

Comment:
WIP in e9b24c1. Doesn’t support functions yet. Sets the bool every iteration, which could lead to inconsistency and should be unnecessary.